packages/e2e-test-app/windows/ReactUWPTestApp/TreeDumpControlViewManager.cs

```diff
@@ -1,365 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-using Microsoft.ReactNative;
-using Microsoft.ReactNative.Managed;
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Text.RegularExpressions;
-using System.Threading.Tasks;
-using Windows.ApplicationModel.ConversationalAgent;
-using Windows.ApplicationModel.DataTransfer;
-using Windows.Data.Json;
-using Windows.Foundation;
-using Windows.Storage;
-using Windows.UI.ViewManagement;
-using Windows.UI.Xaml;
-using Windows.UI.Xaml.Controls;
-using Windows.UI.Xaml.Documents;
-using Windows.UI.Xaml.Media;
-
-namespace TreeDumpLibrary
-{
-    internal class TreeDumpControlViewManager : IViewManager, IViewManagerWithNativeProperties
-    {
-        public string Name => "TreeDumpControl";
-
-        public FrameworkElement CreateView()
-        {
-            m_textBlock = new TextBlock();
-            m_textBlock.TextWrapping = TextWrapping.Wrap;
-            m_textBlock.IsTextSelectionEnabled = false;
-            m_textBlock.LayoutUpdated += async (source, e) =>
-            {
-                await KnownFolders.DocumentsLibrary.CreateFolderAsync("TreeDump", CreationCollisionOption.OpenIfExists);
-                ApplicationView.GetForCurrentView().TryResizeView(new Size(800, 600));
-                var bounds = ApplicationView.GetForCurrentView().VisibleBounds;
-                if (bounds.Width != 800 || bounds.Height != 600)
-                {
-                    // Dump disabled when window size is not 800x600!
-                    UpdateResult(false /*matchDump*/ , $"Window has been resized, dump comparison is only valid at default launch size: 800x600!, current size: {bounds.Width}x{bounds.Height}");
-                }
```

Comment:
this logic is still relevant, the dumps won't match at different resolutions

Reply:
The root view in the window has a fixed size.